src/module/actor/data-model-character.test.js

```diff
@@ -1,60 +1,47 @@
-import OseDataModelCharacter from "./data-model-character"
```

Comment:
missing import?

Reply:
Nope! We're testing the data model by creating a new Actor of type Character.